lms/djangoapps/course_api/tests/test_serializers.py

```diff
@@ -1,9 +1,9 @@
-"""
-Test data created by CourseSerializer and CourseDetailSerializer
-"""
+""" Test data created by CourseSerializer and CourseDetailSerializer """
```

Comment:
Why was this changed?
The standard we have for docstrings is to keep them on separate lines.